plugins/modules/lke_cluster.py

```diff
@@ -225,11 +225,13 @@
             "NOTE: This value may be unavailable if `skip_polling` is true.",
             docs_url="https://techdocs.akamai.com/linode-api/reference/get-lke-cluster-kubeconfig",
             type=FieldType.string,
+            sample=["\"a3ViZWNvbmZpZyBjb250ZW50Cg==\""]
```

Comment:
We'll need a separate ticket to make sure we're parsing the sample according the the type, but this works for now 🙂